maplibre/src/coords.rs

```diff
@@ -460,6 +461,21 @@ fn tiles_with_z(z: u8) -> f64 {
 }
 
 impl WorldCoords {
+    pub fn from_lat_lon(latitude: f64, longitude: f64, zoom: f64) -> WorldCoords {
+        let tile_size = TILE_SIZE * 2.0_f64.powf(zoom);
+        // Get x value
+        let x = (longitude + 180.0) * (tile_size / 360.0);
+
+        // Convert from degrees to radians
+        let lat_rad = (latitude * PI) / 180.0;
+
+        // get y value
+        let merc_n = f64::ln(f64::tan((PI / 4.0) + (lat_rad / 2.0)));
+        let y = (tile_size / 2.0) - (tile_size * merc_n / (2.0 * PI));
+
+        WorldCoords { x, y }
+    }
```

Comment:
Should we already make this more abstract? could this differ between different projections?

Also we should add a type for long/lat.

Reply:
OR: create issue about discussing "projections". Is maplibre-rs projection aware?

Reply:
@Drabble can you create such an issue?

Reply:
#139